pkg/api/vlabs/validate.go

```diff
@@ -274,14 +277,21 @@ func (a *Properties) Validate() error {
 	return nil
 }
 
-func (a *Properties) ValidateNetworkPolicy() error {
-
+func (a *Properties) validateNetworkPolicy() error {
 	if a.OrchestratorProfile.OrchestratorType != Kubernetes {
 		return nil
 	}
 
 	networkPolicy := a.OrchestratorProfile.KubernetesConfig.NetworkPolicy
```

Comment:
Add validation to ensure this only is set when orchestratorType is Kubernetes, and it is a non-windows cluster.